client/templates/home/home.js

```diff
@@ -1,3 +1,4 @@
+Meteor.subscribe("leaderboardSortedTeams");
```

Comment:
@abdelrahman-elkady I couldn't do that because when I do that, the Teams.find({}) in home.js give me the normal Teams. as you are also subscribed to teams. while in my publication i return an aggregated team. so if there's anyway around that tell me. the other way to do it would be to remove the teams subscription.